src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public static final int MAGIC_NUMBER = 47;
```

Comment:
The constant `MAGIC_NUMBER` is not used for any meaningful purpose in your code. It's better to remove it if it's not being used to avoid confusion and to follow the best practices of clean code.